ipalib/util.py

```diff
@@ -1149,3 +1153,40 @@ def no_matching_interface_for_ip_address_warning(addr_list):
                 "{}".format(ip),
                 file=sys.stderr
             )
+
+
+def get_terminal_height(fd=1):
+    """
+    Get current terminal height
+
+    Args:
+        fd (int): file descriptor. Default: 1 (stdout)
+
+    Returns:
+        int: Terminal height
+    """
+    try:
+        return struct.unpack(
+            'hh', fcntl.ioctl(fd, termios.TIOCGWINSZ, b'1234'))[0]
+    except:
```

Comment:
Could you please add an 'Exception' after except? To 'specify' exception type (pylint err).

Reply:
Thanks!

Reply:
No problem :)